# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1346781254 14400
#      Tue Sep 04 13:54:14 2012 -0400
# Branch stable
# Node ID 997e05334f71269f16017ceb06a52cf33feced18
# Parent  a95432e7309ca6fc776c02939264bb6d443f3525
Added tag release-3-6-3 for changeset a95432e7309c

diff --git a/.hgtags b/.hgtags
--- a/.hgtags
+++ b/.hgtags
@@ -58,8 +58,9 @@ ff5588774680d4f54567311fc109c8e351950f1c
 a737b3fb9c4d89d3694da6b4e623aeee64b212e1 ss-3-5-92
 72aebe6196414e38ef802469ff6e238f914f04c9 rc-3-6-0-0
 64d9f33313cc8c691974bcd123357e24bccbabdc rc-3-6-0-1
 704f7895eef03008dd79848eb9da4bfb40787d73 release-3.6.0
 95c43fc8dbe1a07a46fefb3372df5b2309d874fd rc-3-6-1-0
 f947d2922febf12dcd1fb6e21b356756ecb54e55 rc-3-6-2-0
 4460c4fb20e6a5d3b1972fa737d4e00eb921545a rc-3-6-2-2
 551566201318bf615b27c60ccf9368f4844008bd release-3-6-2
+a95432e7309ca6fc776c02939264bb6d443f3525 release-3-6-3
